### PR TITLE
Add webhooks for waiting list events

### PR DESCRIPTION
**Pull request by raphaelm**

### CODE REVIEW

**Review by wiffbi — COMMENTED** [review]
Looks good. What about a webhook, when a waitlinglist-entry is assigned? Is this a change on the waitinglist-entry? Or should we ignore this? Or trigger a webhook on `v.log_action('pretix.voucher.added.waitinglist', …` as voucher-assign happens in at least two places and does the above `log_action` on the voucher – which lends itself to adding webhooks for vouchers as well?

**Comment by raphaelm** [comment]
Thought about this as well but would leave webhooks for vouchers as a future project for now :) That might have some complications with regards to bulk creation

**Comment by wiffbi** [comment]
On the `WaitingListEntry.send_voucher` I already found a fitting `log_action` – `self.log_action('pretix.waitinglist.voucher'` … What about adding this as a `ParametrizedWaitingListEntryWebhookEvent`?

**Comment by wiffbi** [comment]
Yes!

File: src/pretix/control/logdisplay.py [hunk 1]
```diff
@@ -531,9 +531,10 @@ def pretixcontrol_logentry_display(sender: Event, logentry: LogEntry, **kwargs):
         'pretix.event.permissions.invited': _('A user has been invited to the event team.'),
         'pretix.event.permissions.changed': _('A user\'s permissions have been changed.'),
         'pretix.event.permissions.deleted': _('A user has been removed from the event team.'),
-        'pretix.waitinglist.voucher': _('A voucher has been sent to a person on the waiting list.'),
+        'pretix.waitinglist.voucher': _('A voucher has been sent to a person on the waiting list.'),  # legacy
+        'pretix.event.orders.waitinglist.voucher_assigned': _('A voucher has been sent to a person on the waiting list.'),
         'pretix.event.orders.waitinglist.deleted': _('An entry has been removed from the waiting list.'),
-        'pretix.event.orders.waitinglist.transferred': _('An entry has been transferred to another waiting list.'),
+        'pretix.event.orders.waitinglist.transferred': _('An entry has been transferred to another waiting list.'),  # legacy
```
Comment (wiffbi) [inline src/pretix/control/logdisplay.py:537]: There was previously a typo, which made the transferred log-entry not labelled.
```suggestion
        'pretix.event.order.waitinglist.transferred': _('An entry has been transferred to another waiting list.'),  # legacy
```

**Review by wiffbi — APPROVED** [review]
Looks good, just a minor typo that was in there before and is now only used for legacy lag_actions.

**Pull request merged**